.github/workflows/semantic-release.yml

```diff
@@ -0,0 +1,46 @@
+name: Semantic Release
+on:
+  push:
+    branches:
+      - 'master'
+    tags_ignore:
+      - '**'
+  workflow_dispatch:
+
+env:
+  COMMIT_USER: Hiro DevOps
+  COMMIT_EMAIL: 45208873+blockstack-devops@users.noreply.github.com
+
+jobs:
+  semantic-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set Node Version
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: 12.16.1
+      - name: Restore lerna cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+      - name: Code Checks
+        uses: ./.github/actions/code-checks
+      - name: Setup .npmrc
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
+      - name: Setup git
+        run: |
+          git config --global user.email "${COMMIT_EMAIL}"
+          git config --global user.name "${COMMIT_USER}"
+      - name: Lerna Version
+        run: yarn lerna version --conventional-commits --no-push --yes
+      - name: Push changes
```

Comment:
The `lerna version` command above can push the changes if you remove the `--no-push` flag.

Reply:
I think when I tried that many months ago, something happened where I couldn't pass the `GH_TOKEN` to lerna. I probably missed something obvious, but that's why I have the extra step.

Reply:
If you change the `checkout` step to use the `GH_TOKEN`, pushes through `lerna` should work.
```
      - uses: actions/checkout@v2
        with:
          token: ${{ secrets.GH_TOKEN || secrets.GITHUB_TOKEN }}
```